cf_remote/remote.py

```diff
@@ -632,3 +632,29 @@ def deploy_masterfiles(host, tarball, *, connection=None):
         return 1
     print("Policy set successfully deployed to '%s' 🚀" % host)
     return 0
+
+
+@auto_connect
+def open_shell(host, connection=None):
+    print("Enter 'exit' to log out\n")
+
+    cmd = ""
+    while cmd != "exit":
+        try:
+            cmd = input("\033[92m{}\033[00m :~ $ ".format(host))
+            result = connection.run(cmd, hide=True)
+
+            if cmd == "exit":
+                pass
+            elif result.retcode == 0:
+                print(result.stdout.strip().strip("\n"))
+            else:
+                print(result.stderr.strip("\n"))
+        except KeyboardInterrupt:
+            print()
+            continue
+        except EOFError:
+            print()
+            return 0
+
+    return 0
```

Comment:
Did you try just launching `ssh`, instead of "implementing" a new shell like this?

https://stackoverflow.com/questions/3692387/start-interactive-ssh-session-from-python-script

I know it might be a bit tricky to get it to work, but might be worth it if it's not too hard.